app/assets/stylesheets/refinery/stories.scss

```diff
@@ -106,21 +106,98 @@
     vertical-align: center;
   }
 
+  .stories.container {
```

Comment:
In researching this for implementation in vscode, they are recommending a `.sass-lint.yml` config file in the project.  I could do that and then `gitignore` it but wanted to check first if you have something in the project for this already, or are you just implementing the rules within sublime on your end?

Reply:
I put the `.sass-lint.yml` in my home directory which for most editors it should check there if there isn't a project specific one. That being said I think a lot of times folks will check their lint configuration into git for the project because then that way other folks checking out the project know what format they're using.So I'd be ok if you didn't put it in .gitignore, but I put the file above in `~/.sass-lint.yml`